pyctcdecode/tests/test_decoder.py

```diff
@@ -600,3 +600,29 @@ def test_serialization(self):
 
         # this decoder has a language model so we expect one more key
         self.assertEqual(old_num_models + 1, self._count_num_language_models())
+
+    def test_load_from_hub_offline(self):
+        # test local loading
+        self.clear_dir()
+
+        # create language model and decode text
+        alphabet = Alphabet.build_alphabet(SAMPLE_LABELS)
+        language_model = LanguageModel(TEST_KENLM_MODEL, alpha=1.0)
+        decoder = BeamSearchDecoderCTC(alphabet, language_model)
+        text = decoder.decode(TEST_LOGITS)
+        self.assertEqual(text, "bugs bunny")
+
+        # create dummy cached hf dir
+        dummy_hub_name = "kensho/dummy_test"
+        dummy_cached_subdir = dummy_hub_name.replace("/", "__") + ".123456aoeusnth"
```

Comment:
can we add a comment what the `replace` and file extension mean / do here. I confess I'm a bit confused by the file extension